torch/onnx/__init__.py

```diff
@@ -147,6 +147,13 @@ def export(model, args, f, export_params=True, verbose=False, training=False,
             to 1 by default.
         enable_onnx_checker (bool, default True): If True the onnx model checker will be run
             as part of the export, to ensure the exported model is a valid ONNX model.
+        use_large_model_format (bool, default False): If True, then the model is exported
```

Comment:
large model format is a bit misleading. I would suggest external_data_format. And we can provide a link to the description of this format. The format is introduced in https://github.com/onnx/onnx/pull/678, but I didn't find a good link to the dedicated doc (probably we don't have it.). :-)

Reply:
Agreed. I have updated the name everywhere. I have also added a link to the ONNX spec for the external data format here.